### PR TITLE
Add reference to charts-repo-actions-demo

### LINKED ISSUE #241: Practical how to guide
The current documentation is targeted at commands of the application rather than the experience of using it and accomplishing tasks. If I were to point someone here and suggest they start using it they would likely not know where to begin or what to do.

A way to make it easier to onboard people is to provide practical how-to documentation.

This is an issue to record adding such documentation.

### PR DESCRIPTION
**Pull request by nvtkaszpir**
**What this PR does / why we need it**:

Referencing charts-repo-actions-demo in docs should be beneficial for those
looking for more in-depth examples of how to use ct tool.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: 

Fixes #241

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by nvtkaszpir** [comment]
ping @rimusz @unguiculus

**Review by davidkarlsen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -51,6 +51,9 @@ See documentation for individual commands:
 * [ct list-changed](doc/ct_list-changed.md)
 * [ct version](doc/ct_version.md)
 
+For more extensive how-to guide it is recommended to see:
```
Comment (unguiculus) [inline README.md:54]: I'd suggest to reword this:

```suggestion
For a more extensive how-to guide, please see:

```

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Comment by nvtkaszpir** [comment]
slight ping towards @unguiculus

**Review by unguiculus — APPROVED** [review]

**Pull request merged**